protocol/network/messages.md

```diff
@@ -43,7 +44,9 @@ Commands that are shorter than 12 bytes are right-padded with null bytes (`0x00`
 The command string is used to determine the type of message being transmitted.
 Messages with an unrecognized `command string` are ignored by most implementations but may result in a ban by implementations that diverge from the Satoshi-client defacto standard.
 
-The following messages are considered standard by all node implementations.
+The following messages are considered standard by all node implementations.  
```

Comment:
These are good points, but I don't think we should leave TODOs in the document (unless on branches).  I don't think we need to document when these messages came into the protocol, for pre-fork messages.  So how about:

The following messages are considered standard by all node implementations, as of protocol version 80003 (see [VERSION](/protocol/network/messages/version.md)).

If these standard messages are not implemented, or are improperly implemented, it is possible that a client will be dropped or banned.